src/super_gradients/recipes/cityscapes_kd_base.yaml

```diff
@@ -55,7 +55,7 @@ training_hyperparams:
     weights: [ 1. ]
     kd_loss_weights: [1., 6.]
 
-    KDLogitsLoss:
+    kd_loss:
```

Comment:
No in fact KDLogitsLoss is the right name. `kd_loss` is deprecated name, see https://github.com/Deci-AI/super-gradients/blob/0b79e68b58e12d9aa2d7be5198c1d5016c6fe848/src/super_gradients/training/losses/kd_losses.py#L20